host/src/README.md

```diff
@@ -1,11 +1,7 @@
+# FunctionsNetHost
 
-This is the project root where we have our root level CMakeLists.txt.
-### Load & Build. 
+Managed code version of FunctionsNetHost.
```

Comment:
This comment won't make a lot of sense in the future. 

For now, simply describing this as the "Azure Functions Custom .NET Host" is sufficient. We'll update

Reply:
Flagging to address this in the next iteration. Merging this version now.